Dockerfile

```diff
@@ -6,30 +6,40 @@ FROM rust:${RUST_VERSION}-slim-bookworm as build
 USER root
 
 RUN apt update \
-    && apt install --yes pkg-config libssl-dev build-essential libsqlite3-dev cmake protobuf-compiler \
+    && apt install --yes pkg-config libssl-dev build-essential libsqlite3-dev cmake protobuf-compiler unixodbc-dev \
```

Comment:
For simplicity I keep `unixodbc-dev` always installed on build machine, let me know if we should make it conditional.